docs/app/Examples/modules/Popup/Types/PopupHtmlExample.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Popup, Card, Rating, Image } from 'stardust'
+
+const users = [
```

Comment:
Ah, I believe I'm getting confused on the new GH review flow.  This comment thread does not show it in use for some reason.

https://github.com/debrice/stardust/blob/d51317361c15723d2796c85a1752ee4d5fc63584/docs/app/Examples/modules/Popup/Types/PopupHtmlExample.js